curriculum/challenges/english/07-scientific-computing-with-python/learn-interfaces-by-building-an-equation-solver/662bd456896f16d9bd03f1a6.md

```diff
@@ -0,0 +1,47 @@
+---
+id: 662bd456896f16d9bd03f1a6
+title: Step 2
+challengeType: 20
+dashedName: step-2
+---
+
+# --description--
+
+Within the `Equation` class, define two new instance methods named `solve` and `analyze`.
```

Comment:
Have we stopped testing if the user has removed the `pass` keyword?
Noticed the same in step-10.

Reply:
For the new additions to the curriculum we are not testing it. Probably, after the first projects, we should stop asking to add `pass` and then replace `pass` with `<x>`... It breaks the flow and complicate the instructions. It has no effect, so once campers understand why we are using it, I think there's no point in asking them to remove it.